app/src/main/assets/instructions.csv

```diff
@@ -34,3 +34,6 @@
 33;Tapping;finger1;Finger tapping 1;The bug game with one finger;Please, Tap as many bugs you can until the time is over. Use only **one** finger. Press Start button when you are ready.;Fingertippen;Fingertippen beschreibung;Fingertippen anweisung;Destreza 1;Destreza con un dedo;A continuación aplaste los insectos que aparecerán en la pantalla utilizando **un sólo dedo** (Mano izquierda **o** derecha). Aplaste tantos insectos le sea posible antes de que se acabe el tiempo. Cuando este listo, presione el botón Empezar ejercicio.;_tapping_one.webm
 34;Tapping;finger2;Finger tapping 2;The bug game with two fingers;Please, Tap as many bugs you can until the time is over. Use **two** fingers. Press Start button when you are ready.;Fingertippen2;Fingertippen2 beschreibung;Fingertippen2 anweisung;Destreza 2;Destreza con dos dedos;A continuación aplaste los insectos que aparecerán en la pantalla utilizando **los dos dedos**. Aplaste tantos insectos le sea posible antes de que se acabe el tiempo. Cuando este listo, presione el botón Empezar ejercicio.;_tapping_two.webm
 35;Tapping;sliding;Sliding;Move the bar;Please move the bar to the target position. Press Start exercise when you are ready.;Ruschen;Ruschen beschreibung;Ruschen anweisung;Desplazamineto de barra;Mover la barra;Mueva la barra que aparecerá en pantalla hasta la posición indicada. Cuando este listo, presione el botón Empezar ejercicio.;_sliding.webm
+36;Facial;rightwink;RightWink;Winks the eye;Please winks the right eye;RightWink;Winks the eye;Please winks the right eye;Guiñar el ojo;Guiña el ojo derecho;Por favor guiña el ojo derecho como lo muestra el emoticon;_sliding.webm
+37;Facial;leftwink;leftWink;Winks the eye;Please winks the left eye;leftWink;Winks the eye;Please winks the left eye;Guiñar el ojo;Guiña el ojo izquierdo;Por favor guiña el ojo izquierdo como lo muestra el emoticon;_sliding.webm
```

Comment:
If you dont have the proper video instructions for this, remove the video instruction for the sliding